installers/release-notes.gradle

```diff
@@ -42,11 +42,11 @@ task releaseNotes {
 
     ByteArrayOutputStream gitContributorsOut = new ByteArrayOutputStream()
     exec {
-      commandLine "git", "log", "--no-merges", "--format=%cN", "16.6.0..${rootProject.gitRevision}"
+      commandLine "bash", '-c', "git shortlog -s --no-merges 16.6.0..${rootProject.gitRevision} | awk '!(\$1=\"\")' | tr '\n' ',' | sed s'/.\$//' | sed s'/^.//'"
```

Comment:
Another reason being that errors in piping are not quite captured and bubbled out properly. The following returns a `0` exit code 

`true | false | true`

whereas this returns a non-zero —

`true | true | false`